src/modules/launcher/PowerLauncher.Telemetry/Events/PluginModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+
+namespace PowerLauncher.Telemetry.Events
+{
+    [EventData]
+    public class PluginModel
+    {
+        public string Name { get; set; }
```

Comment:
Let's remove the `Name` property since this information is already present in the JSON object.

Reply:
done